theme/theme.go

```diff
@@ -41,6 +42,7 @@ func DarkTheme() fyne.Theme {
 		background:  color.RGBA{0x42, 0x42, 0x42, 0xff},
 		button:      color.RGBA{0x21, 0x21, 0x21, 0xff},
 		text:        color.RGBA{0xff, 0xff, 0xff, 0xff},
+		hyperlink:   color.RGBA{0x0, 0x0, 0xff, 0xdd},
```

Comment:
Is it correct that the blue is the same in light and dark themes?

Reply:
Yeah, feel free to modify that if you want.  In no way do you want me designing colors :fearful: I tested in both the light and dark theme and it looked okay to me, but my feelings won't be hurt if you change it to something else!